### PR TITLE
Fix chat styling; fix focus issues in chat

### PR DESCRIPTION
**Pull request by somebody1234**
### Pull Request Description
- Implement https://github.com/enso-org/cloud-v2/issues/885
  - Adjust chat styling to match more with the rest of the `ide2` Figma design (and the screenshots from the original issue...)

### Important Notes
None

### Screenshots
Old chat styling (obsoleted by this PR):
image

New chat styling (in drive view):
image

New chat styling (in IDE1):
image

New chat styling (in IDE2):
image

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] ~~The documentation has been updated, if necessary.~~
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] ~~Unit tests have been written where possible.~~
  - [x] ~~If GUI codebase was changed, the GUI was tested when built using `./run ide build`.~~

### CODE REVIEW

**Comment by somebody1234** [comment]
pinging @wdanilo mostly (?) as a heads up

File: app/ide-desktop/lib/common/src/detect.ts [hunk 1]
```diff
@@ -7,6 +7,20 @@
...
+// ====================
+
+/** Whether the current IDE being used is GUI1. */
+export function isGUI1() {
+    return document.getElementById('root') != null
+}
+
+/** Whether the current IDE being used is GUI2. */
+export function isGUI2() {
+    return document.getElementById('root') != null
```
Comment (PabloBuchu) [inline app/ide-desktop/lib/common/src/detect.ts]: so how exactly does it work since isGUI1 and isGUI2 always returns the same?

**Comment by PabloBuchu** [comment]
is it just a beginning of changes or it was just a fix? Not sure what new design you mean and the issue is also vague

File: app/ide-desktop/lib/common/src/detect.ts (hunk 1 above)
Comment (somebody1234) [reply app/ide-desktop/lib/common/src/detect.ts]: oops... will fix asap

**Comment by somebody1234** [comment]
@PabloBuchu sorry, my bad - this should be all the changes.
- re: the new design - i don't believe there is one for chat. i just think it looks out of place with a white background, when everything else is more or less transparent
  - but also: i guess "new design" is a bad way to put it - "current design" probably makes more sense, as opposed to the very old design that was phased out quite a while ago

**Comment by somebody1234** [comment]
oh, actually chat was never close to the original design...
- https://github.com/enso-org/enso/issues/6610

so this PR actually brings chat closer to the *initial* design

**Review by PabloBuchu — APPROVED** [review]

**Pull request merged**